### ISSUE TITLE
FPP Connect and Falcon F16v4

### ISSUE THREAD

**Issue by jnealand** [smeighan/xLights#4209]
When using FPP connect with my two Falcon F16v4s the brightness setting is always changed to 100% for every port being uploaded.  I always set my brightness in the controllers and never using the brightness setting on each model.  Personally I set all my controllers to 30% for each port.  I have F16v2, F16v3 and F16v4 in my show and only the F16v4s seem to have this issue.

**Comment by computergeek1507** [comment]
Do you have full xLights enabled and the default brightness set?

**Comment by jnealand** [comment]
I do not use full xlights control as I do not understand what that is and what it might do to my display.  New things have killed me in the past so I avoid them until I have to learn about them.

I did not even know there was a place to set default brightness or/if that I had to do that.  In previous years we have always been told to set the brightnes in only one spot and for years I have done that always in the the controller.
